apps/frontend/src/store/snackbar.ts

```diff
@@ -0,0 +1,48 @@
+import {Module, VuexModule, Mutation, getModule} from 'vuex-module-decorators';
+import Store from '@/store/store';
+
+export interface ISnackbarState {
+  message: string;
+  error: boolean;
+  show: boolean;
+}
+
+@Module({
+  namespaced: true,
+  dynamic: true,
+  store: Store,
+  name: 'SnackbarModule'
+})
+export class Snackbar extends VuexModule {
+  message = '';
+  error = false;
+  show = false;
+  @Mutation
+  success(message: string) {
+    this.error = false;
+    this.message = message;
+    this.show = true;
+  }
+  @Mutation
+  failure(message: string) {
+    this.error = true;
+    this.message = message;
+    this.show = true;
+  }
```

Comment:
These should be `@Action` not `@Mutation` and they should call the mutations and not set the values directly.